### PR TITLE
[Keyboard] Add VIA keymap for the reviung39

### PR DESCRIPTION
**Pull request by kbrock**
## Description

I based this upon the reviung41 via keymap that is in our code base but used `reviung39/keymaps/default` as the base instead.

Thanks.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/reviung39/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    _RAISE,
+    _ADJUST
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT_reviung41(
+    KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,      KC_T,               KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_BSPC,
+    KC_LCTL,  KC_A,     KC_S,     KC_D,     KC_F,      KC_G,               KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,
+    KC_LSFT,  KC_Z,     KC_X,     KC_C,     KC_V,      KC_B,               KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  RSFT_T(KC_ENT),
+                                                       MO(1),    KC_SPC,   MO(2)
```
Comment (drashna) [inline keyboards/reviung39/keymaps/via/keymap.c:30]:
```suggestion
                                                       FN_MO13, KC_SPC,   FN_MO23
```

Comment (kbrock) [reply keyboards/reviung39/keymaps/via/keymap.c]: @drashna thank you

**Review by drashna — APPROVED** [review]

**Comment by kbrock** [comment]
Thanks for pointing me to `FN_MO13 `. This is my first `via` port and I am not familiar with that syntax.

Reviung41 has a key for layer 3, but the reviung39 (this one) relies upon pressing raise/lower for the adjust layer. Do we still the `tri_state_layer` for via or is that handled once I update the via firmware?

 ```c
layer_state_t layer_state_set_user(layer_state_t state) {
  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
}
```

**Comment by drashna** [comment]
The keycodes I mentioned specifically handles the tri layer stuff itself, no need to have the layer_state_set_user call. In fact, you want one or the other, not both.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**